mautic/mautic
# [UI] Enhance page body

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR fixes the grid system by applying the indicated structure in the [Bootstrap v3.4.1 documentation](https://getbootstrap.com/docs/3.4/css/#grid).

> Bootstrap requires a containing element to wrap site contents and house our grid system.

> Rows must be placed within a .container (fixed-width) or .container-fluid (full-width) for proper alignment and padding.

In a practical scenario, it increases a bit of the body padding and avoid bugs with new components. 
I applied the full-width container to ensure a standard experience later.

It also reduces the top navbar size from 60px to 48 pixels.

This allows users to focus on the most important information without being distracted by too many elements piled up on the screen. In addition, more white space can make the interface more pleasing to the eyes.

| Before | After
| -------------------------------------- | ---
| ![image](https://github.com/mautic/mautic/assets/116097999/f307d33e-b7d2-4bd8-9ecc-75ff28587e42)   | ![image](https://github.com/mautic/mautic/assets/116097999/a7b66dd9-604b-481d-a8f2-b7298a1e95a5)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You can open any page to see the space (16px horizontal for the body content)
3. Look at the top navbar to see the height reduction
